dsf-docker/fhir_proxy/Dockerfile

```diff
@@ -9,4 +9,14 @@ RUN mkdir /usr/local/apache2/ssl/ && \
 
 # setting non existing default values, see host-ssl.conf IfFile tests
 ENV SSL_CERTIFICATE_CHAIN_FILE="/does/not/exist"
-ENV SSL_CA_DN_REQUEST_FILE="/does/not/exist"
\ No newline at end of file
+ENV SSL_CA_DN_REQUEST_FILE="/does/not/exist"
+
+# timeout (seconds) for reverse proxy to app server http connection, time the proxy waits for a reply
+ENV PROXY_PASS_HTTP_TIMEOUT=60
+# timeout (seconds) for reverse proxy to app server ws connection, time the proxy waits for a reply
+ENV PROXY_PASS_WS_TIMEOUT=60

```

Comment:
In all our property values we use a string that is as equal as possible at the beginning and has the differences at the end. This is more convenient when using them. Therefore maybe use `PROXY_PASS_TIMEOUT_HTTP` and `PROXY_PASS_TIMEOUT_WS`, same for the `CONNECTION_TIMEOUT`.

Reply:
Good point. I'll change them.